pkgs/applications/networking/onionshare/stem.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildPythonPackage, fetchFromGitHub, python, mock }:
+
+buildPythonPackage rec {
```

Comment:
Shouldn't this be under pythonPackages?

Reply:
It should be and it is but Onionshare depends on its customized fork of that module. The normal stem from Tor Project doesn't work. (for now)

Reply:
Curious why you didn't override the existing package for this though.,

Reply:
I thought it wouldn't work, but actually it does so there you go.